clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -1186,6 +1200,18 @@ public OffsetAndMetadata committed(TopicPartition partition) {
      */
     @Override
     public void pause(TopicPartition... partitions) {
+        pause(Arrays.asList(partitions));
+    }
+
+    /**
+     * Suspend fetching from the requested partitions. Future calls to {@link #poll(long)} will not return
```

Comment:
Maybe instead of repeating the full javadoc, we can just have a short description with a link to the other variant. For example: 
```java
/**
 * Pause the specified partitions. See {@link #pause(TopicPartition...)} for more detail.
 * @param partitions The partitions which should be paused
 */
```